tests/unit-tests/devfile-page/Header.test.tsx

```diff
@@ -0,0 +1,184 @@
+import { Devfile } from 'custom-types';
+import Header from '../../../components/devfile-page/Header';
```

Comment:
Im guessing the `@components` still doesnt work?

Reply:
so it worked for a hot second on one specific test file (Projects.test.tsx) but its not working on any of them anymore -- i think this one specific path (@components/devfile-page/Projects) was cached for a bit on my computer, and if that was why this one path was working, then it's not reliable
I also tried taking the unit-tests out of the tsconfig exclude files but that didn't fix the problem either, just made vscode pop up with a bunch of syntax errors